src/main/java/janggi/domain/move/Vector.java

```diff
@@ -0,0 +1,21 @@
+package janggi.domain.move;
+
+import janggi.domain.Side;
+
+public record Vector(int y, int x) {
+
+    public static Vector rotate(Vector vector) {
+        return new Vector(vector.x(), -vector.y());
+    }
+
+    public Vector side(Side side) {
+        if (side == Side.CHO) {
```

Comment:
side가 CHO 인지는 side가 판단하는 것이 어떨까요?
```suggestion
        if (side.isCho()) {
```